rlw.sh

```diff
@@ -259,7 +265,7 @@ fi
 
 # Note: git is used for automatic updating, and is recommended.
 [[ -x "$(which git)" ]] || {
-	spawndialog error "git is not installed, or was not found. Please install git\nto enable automatic updates."
+	spawndialog error "git is not installed, or was not found. Please install git\nto enable automatic updates./nsudo apt-get install git "
```

Comment:
This isn't quite consistent with the other dependency checking.. Remember, not everyone uses Debian/Ubuntu or a derivative.